ament_uncrustify/ament_uncrustify/main.py

```diff
@@ -54,12 +56,18 @@ def main(argv=sys.argv[1:]):
         default=[os.curdir],
         help='The files or directories to check. For directories files ending '
              'in %s will be considered.' %
-             ', '.join(["'.%s'" % e for e in extensions]))
+             ', '.join([
+                 "'.%s'" % e for e in sorted(c_extensions + cpp_extensions)]))
     parser.add_argument(
         '--exclude',
         nargs='*',
         default=[],
         help='Exclude specific file names and directory names from the check')
+    parser.add_argument(
+        '--language',
+        choices=['C', 'C++'],
+        help="Passed to uncrustify as '-l <language>' to force a specific "
+             'rather then choose based on the file extensions')
```

Comment:
@dirk-thomas nit: 
```suggestion
             'rather than choosing one based on file extension')
```

Reply:
Thanks, done in 7f015e5394989595167130a71c649eface660017 to balance the line length.